src/sentinel.c

```diff
@@ -3331,12 +3377,12 @@ int sentinelSendSlaveOf(sentinelRedisInstance *ri, char *host, int port) {
     ri->pending_commands++;
 
     retval = redisAsyncCommand(ri->cc,
-        sentinelDiscardReplyCallback, NULL, "SLAVEOF %s %s", host, portstr);
+        sentinelDiscardReplyCallback, NULL, "%s %s %s", ri->master->slave_of_command, host, portstr);
```

Comment:
I don't see how it could be NULL. I set the default in `createSentinelRedisInstance` [here] (https://github.com/jipperinbham/redis/blob/sentinel-rename-commands/src/sentinel.c#L993-L995) which then gets applied to the master and is called from `sentinelHandleConfiguration` [here](https://github.com/jipperinbham/redis/blob/sentinel-rename-commands/src/sentinel.c#L1351-L1352). Then, if `rename-*` command is specified in the config, it gets applied to the master [here](https://github.com/jipperinbham/redis/blob/sentinel-rename-commands/src/sentinel.c#L1400-L1409).